package.json

```diff
@@ -21,11 +21,8 @@
     "url": "https://github.com/biancadanforth/tracking-protection-shield-study/issues"
   },
   "devDependencies": {
-    "addons-linter": "^0.28.2",
     "ajv": "^5.1.1",
-    "clipboardy": "^1.1.4",
```

Comment:
This doesn't seem super useful to me. Yes it's from the template. I can access my clipboard just fine, thanks?